.vscode/launch.json

```diff
@@ -105,6 +90,20 @@
             ],
             "sourceMaps": true,
             "outDir": "${workspaceRoot}/out/"
+        },
+        {
+            "name": "Launch Tests",
+            "type": "extensionHost",
+            "request": "launch",
+            "runtimeExecutable": "${execPath}",
+            "args": [
+                "--extensionDevelopmentPath=${workspaceRoot}",
+                "--extensionTestsPath=${workspaceRoot}/out/test"
```

Comment:
"test" is a convention in most test frameworks, I think we should leave it like that